frontend/lib/src/components/elements/Popover/Popover.tsx

```diff
@@ -144,8 +144,8 @@ const Popover: React.FC<React.PropsWithChildren<PopoverProps>> = ({
                   color="inherit"
                   aria-hidden="true"
                   size="lg"
-                  margin=""
-                  padding=""
+                  margin="0"
+                  padding="0"
```

Comment:
Could we also use `theme.spacing.none` here?

Reply:
Changed 👍